src/main/python/pythonGUI/createStudyAndScenario.py

```diff
@@ -84,8 +84,8 @@ def __init__(self, root, emme_version = "4.3.7", year = "2016", geo = "1", lu =
 
             #divider line
             divider=u"_"*200
-            self.releaseDir='T:\\ABM\\release\\ABM'
-            self.defaultScenarioDir="T:\\projects\\sr14"
+            self.releaseDir=os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..', '..'))

```

Comment:
Should this be an extra argument in the GUI? This script will likely be compiled into an executable file, so I'm not sure that this will work

Reply:
This could be an argument in the GUI, but I think it would be more user friendly if it could get the path from the executable location. I think os.path.dirname(sys.executable) should work for getting the executable path, with a check for if it is being run as a python script or an executable.

Reply:
Added check for compiled executable